attribute.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+# Represents a classroom.
+class Classroom
+  attr_accessor :label, :students
+
```

Comment:
- [ ] The `students` instance variable should not be modified from the outside so as such I'd suggest placing it in an `attr_reader`.